.drone.star

```diff
@@ -192,12 +197,7 @@ def yarnlint():
 			lintTest(),
 		'depends_on': [],
 		'trigger': {
-			'ref': [
-				'refs/tags/**',
-				'refs/pull/**',
-				'refs/pull-requests/**',
-				'refs/merge-requests/**',
-			]
+			'ref': list(config['trigger']['ref'])
 		}
```

Comment:
Can this be just:
```
'trigger': config['trigger']
```

Reply:
@phil-davis this part is later used below in
```
	for branch in config['branches']:
		result['trigger']['ref'].append('refs/heads/%s' % branch)
```
This gives error because it cannot append to the list because it is frozen.